### PR TITLE
Adds build script for integration-testing locally

### PR DESCRIPTION
**Pull request by ckartik**

### CODE REVIEW

File: build-platform.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    start)
+        rpc_url=${2:-$DEFAULT_RPC_URL}  # Use the provided RPC URL or default if not provided
+        datadog_key=${3:-""}  # Use the provided Datadog key or empty if not provided
+        start_settlement_layer
+        deploy_contracts "$rpc_url"
+
+        # Pause for user to update config files
+        echo "Please update /integration-test/config/{nodetype}.yml files as needed."
+        echo "Press Enter to continue after you have made the updates..."
+        echo "Note: If you're on mac, set RPC endpoint to host.internal.docker to connect to localhost"
```
Comment (kant777) [inline build-platform.sh:76]: you can get the os info and set it

Comment (kant777) [reply build-platform.sh:76]: not required though.

File: build-platform.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Default RPC URL
+DEFAULT_RPC_URL="http://localhost:8545"
+
+# Function to start the settlement layer
+start_settlement_layer() {
+    cd ~/
+    git clone git@github.com:primevprotocol/go-ethereum.git
```
Comment (shaspitz) [inline build-platform.sh:9]: Cloning geth from top level like this defeats the purpose of having the docker setup live within geth repo, maybe suggests monorepo would be appropriate in medium/long term

**Review by shaspitz — APPROVED** [review]

File: build-platform.sh (hunk 1 above)
Comment (kant777) [reply build-platform.sh:76]: sounds good!

File: build-platform.sh (hunk 2 above)
Comment (kant777) [reply build-platform.sh:9]: @shaspitz Agreed. However, this might be ok for now until we have the artifact repository where we can push all our binaries.

**Pull request merged**